File: WPF客户端/AnnHome/AnnHome/Entity/Attachments.cs
using System;
using System.Collections.Generic;

namespace AnnHome.Entity
{
    [Serializable]
    internal class Attachments:List<Object>
    {
        private int id;
        private string url;
        private string slug;
        private string title;
        private string description;
        private string caption;
        private int parent;
        private string mime_type;
        private Images images;

        public int Id
        {
            get
            {
                return id;
            }

            set
            {
                id = value;
            }
        }

        public string Url
        {
            get
            {
                return url;
            }

            set
            {
                url = value;
            }
        }

        public string Slug
        {
            get
            {
                return slug;
            }

            set
            {
                slug = value;
            }
        }

        public string Title
        {
            get
            {
                return title;
            }

            set
            {
                title = value;
            }
        }

        public string Description
        {
            get
            {
                return description;
            }

            set
            {
                description = value;
            }
        }

        public string Caption
        {
            get
            {
                return caption;
            }

            set
            {
                caption = value;
            }
        }

        public int Parent
        {
            get
            {
                return parent;
            }

            set
            {
                parent = value;
            }
        }

        public string Mime_type
        {
            get
            {
                return mime_type;
            }

            set
            {
                mime_type = value;
            }
        }

        public Images Images
        {
            get
            {
                return images;
            }

            set
            {
                images = value;
            }
        }
    }
}

File: Android客户端/AnnHome/app/src/main/java/entity/Attachments.java
package entity;

import java.io.Serializable;

/**
 * Created by mrpan on 15/11/17.
 */
public class Attachments extends BaseEntity implements Serializable {
    private static final long serialVersionUID = 1L;

    private int id;
    private String url;
    private String slug;
    private String title;
    private String description;
    private String caption;
    private int parent;
    private String mime_type;
    private Images images;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getCaption() {
        return caption;
    }

    public void setCaption(String caption) {
        this.caption = caption;
    }

    public int getParent() {
        return parent;
    }

    public void setParent(int parent) {
        this.parent = parent;
    }

    public String getMime_type() {
        return mime_type;
    }

    public void setMime_type(String mime_type) {
        this.mime_type = mime_type;
    }

    public Images getImages() {
        return images;
    }

    public void setImages(Images images) {
        this.images = images;
    }
}
